manim/mobject/matrix.py

```diff
@@ -96,7 +96,11 @@ def __init__(
         self.left_bracket = left_bracket
         self.right_bracket = right_bracket
         VMobject.__init__(self, **kwargs)
-        matrix = np.array(matrix, ndmin=1)
+        matrix = np.array(matrix)
+        if len(matrix.shape) < 2:
+            raise ValueError(
+                f"{self.__str__()} class requires a two-dimensional array!"
+            )
```

Comment:
Nitpick, non-blocking: if you check the dimension before creating an array, you would save some work in case the dimension is wrong. For example if a Matrix is initialized with a very long list of elements, there is no reason to create the array just to error out immediately. So my suggestion is to invert these two statements :)

Reply:
Good point. I updated the code.